scripts/azure-pipelines/bootstrap-from-source.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
```

Comment:
```suggestion
set -e

```

Otherwise, failure to checkout, failure to configure, failure to build will all not be diagnosed correctly.